src/client/datascience/jupyter/kernels/cellExecution.ts

```diff
@@ -279,18 +277,15 @@ export class CellExecution {
         }
 
         // If there are any errors in the cell, then change status to error.
-        if (hasErrorOutput(this.cell.outputs)) {
+        if (this.cell.outputs.some(isErrorOutput)) {
```

Comment:
the `hasErrorOutput` method was a single liner now after refactoring, hence removed that one line function.